### PR TITLE
WiP

### PR DESCRIPTION
**Pull request by MrElectroNick**
Please note that I made gulp tasks to run in es5 mode - this is because I'm on Mint 17.3 with default node (v0.12) due to work dependencies

### CODE REVIEW

**Comment by sirikon** [comment]
@MrElectroNick Cool! 👍 Just a few things:
- Why did you disabled the sass linting? If you're not okay with the linting configuration we can discuss it to use a better one, but we should _have_ one.
- Also, about downgrading the ES5 the node files, I would like to start using the most recent Node versions (because Node 7 is the new LTS and so) and I don't want to get the project limited at the start by the packages only available to old Node versions, please, check if you can use a newer node version using something like [nvm](https://github.com/creationix/nvm) (With nvm you can have multiple node versions installed in your machine, and change it using a single command: `nvm use 0.12.0` or `nvm use 7.0.0`
- About the mountain of spices:
  - We should use SVG format for all the possible image resources in the webpage, this way we have always better rendering and lower bandwidth usage for the users visiting cinnamon spices. So, could you use a SVG version instead?
  - Anyway, the little spices on top of the mountain should be animated, and be continuously falling into the mountain, this was something that was told in Slack and doesn't wrote it down anywhere... my bad.

Please, check if you can resolve somehow the first two points, If you doesn't have a proper vector graphics software I can do the SVG mountain with its animations later :)

Thank you very much for your efforts! 👍

File: src/style/components/_items-list.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      .overlay {
+        opacity: 0.75;
+      }
+      img {
+        opacity: 1;
+      }
+    }
+  }
+
+  .image {
```
Comment (sirikon) [inline src/style/components/_items-list.scss:33]: Here and a bit down you're using plain classes like 'image', 'overlay'... We should respect here the BEM naming, establishing a new component name or continuing with it's current parent.

For example: cs-items-list-item-image, cs-items-list-item-image-overlay and so.

Comment (MrElectroNick) [reply src/style/components/_items-list.scss:33]: BEM actually accepts isolated classes inside namespaced, class image wouldn't work outside of cs-items-list-item. In any case it's not an issue for such a small project like this

**Comment by MrElectroNick** [comment]
es5/es6 - I can try nvm - never used it before, if it is context (session) independent then it's possible, otherwise some workers running on node on my machine would fail :-)
svg - I work on Mint 17.3 for now and use Gimp - didn't found a way to extract svg yet but I agree - we should use svg instead of png - please do the change if you have a way to make that mountain in svg

File: src/style/components/_items-list.scss (hunk 1 above)
Comment (sirikon) [reply src/style/components/_items-list.scss:33]: The preocupation is more with the possibilitty of other code using the .image class and provoking unexpected behaviours, this is bad by my own experience...

But something true is that this project is pretty controlled and with well-known dependencies (This is my objective, at least).

Do whatever you want with this :)

**Comment by sirikon** [comment]
@MrElectroNick You can set a default node version which will be loaded every time the session starts, but you can change it whenever you want.

So, for example set 0.12 as default, and then invoke 'nvm use 7.0.0' whenever you need it. Once the command line session is closed, node version will be again 0.12 the next session.

If you re-enable the sass linting (or discuss a better one) and try NVM to use newer node version (and syntax) I can merge the PR :)

**Comment by nyrrad** [comment]
I have a concern about some of the class names. I'm seeing cs-button and btn-x-x for example, We should decide if we want to use abbreviations or long case names to be consistent.

**Comment by sirikon** [comment]
@nyrrad nice point... I would vote for long case names

**Comment by nyrrad** [comment]
@Sirikon works for me. How about you @MrElectroNick ?

**Comment by sirikon** [comment]
Really nice work @MrElectroNick ;D

Merging!

**Pull request merged**